cgo/pow/pow.go

```diff
@@ -71,8 +72,9 @@ func DcurlEntry(trytes Trytes, mwm int, parallelism ...int) (Trytes, error) {
 	}
 
 	returnTrytesC := C.dcurl_wrapper(C.CString(trytes), C.int(mwm), C.int(numThread))
-	returnTrytesGO := C.GoString(returnTrytesC)
-	nonce := returnTrytesGO[len(returnTrytesGO)-NonceTrinarySize/3-1 : len(returnTrytesGO)-1]
+	returnTrytesGO := C.GoString(returnTrytesC)[:2673]
```

Comment:
Add comments for the following constants.

Reply:
I resolved the constant 2673 in line 75 to `TransactionTrytesSize` from [iota.go/consts](https://github.com/iotaledger/iota.go/blob/3dfaebe62d8b4037de292cd9c0bbd1a6027a9873/consts/consts.go#L133) for it is more concise than adding comment.

And, I added comment for the fallowing behavior of extracting nonce from trytes.